docs/docs/reference/release-notes/v3.4/index.md

```diff
@@ -0,0 +1,35 @@
+---
+date: "2021-04-27"
+version: "3.4.0"
+---
+
+# [v3.4](https://github.com/gatsbyjs/gatsby/compare/gatsby@3.4.0-next.0...gatsby@3.4.0) (April 2021 #2)
+
+Welcome to `gatsby@3.4.0` release (April 2021 #2)
+
+Key highlights of this release:
+
+- [TODO](#todo)
+
+Also check out [notable bugfixes](#notable-bugfixes--improvements).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v3.3)
+
+[Full changelog](https://github.com/gatsbyjs/gatsby/compare/gatsby@3.4.0-next.0...gatsby@3.4.0)
+
+---
+
+TODO
```

Comment:
```suggestion
## Experimental: Enable webpack persistent caching for production builds

[Webpack 5 introduced built in persistent caching](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#persistent-caching). It allows webpack to reuse result of previous compilations and significantly speed up compilation steps.

To use it, add a flag to your `gatsby-config.js`:

```js
// In your gatsby-config.js
module.exports = {
  // your existing config
  flags: {
    PRESERVE_WEBPACK_CACHE: true,
  },
}
```

[Details and discussion](https://github.com/gatsbyjs/gatsby/discussions/28331).
```
(note that discussion will need to be updated, but I'm not updating it yet as we didn't release addition of persistent caching yet in stable for the flag)